modules/react-map-gl-draw/src/mode-handler.tsx

```diff
@@ -308,12 +307,6 @@ export default class ModeHandler extends React.PureComponent<EditorProps, Editor
     this._modeHandler.handleClick(event, modeProps);
   };
 
-  _onDblclick = (event: BaseEvent) => {
```

Comment:
react-map-gl-draw is not using deck.gl as renderer. so this should be kept.

Reply:
Got it. I'll revert my changes in the `react-map-gl-draw` module.